docs/migrate-guide-0.5.md

```diff
@@ -74,7 +74,7 @@ Will not break old codes but will throw deprecation warning.
 1. Deprecated `DGLGraph` methods:
```

Comment:
This is migration guide for DGL v0.5. I don't think you should modify this file.